foxglove/shared/ros-typescript-generator/README.md

```diff
@@ -0,0 +1,36 @@
+# ROS TypeScript Generator
+A CLI for generating typescript interfaces and enums from ros `.msg`, `.srv`, and `.action` files.
+
+This package has been modified to provide support for the conversion of custom messages without ROS message dependencies. The [original package](https://github.com/Greenroom-Robotics/ros-typescript-generator) was created by Greenroom Robotics.
```

Comment:
Also mention the dependencies that we upgraded to enable the necessary modifications.